docs/content/topics/administrators/authorization.md

```diff
@@ -11,4 +11,117 @@ aliases:
 
 # Brigade Authorization
 
-TODO
+Authorization in Brigade consists of roles with particular scopes, which are
+granted to users and service accounts. When users interact with Brigade via
+the brig CLI or when a service account interacts with Brigade via an SDK,
+Brigade checks to be sure the requestor is sufficiently authorized before
+proceeding.
+
+The three core authorization components in Brigade are:
+
+  * [Users](#users)
+  * [Service Accounts](#service-accounts)
+  * [Roles](#roles)
+
+Users are generated in the system after successful authentication with the
+selected third-party auth provider and the creation of service accounts and
+role assignments is the responsibility of the administrator of Brigade.
+
+Note: There is one method of auto-creation of role assignments for a given set
```

Comment:
nit: do you want to make this an actual markdown note?

Reply:
I think this is how we've generally done "notes" elsewhere, but now I'm curious...

How is a proper note done in markdown? Searching for this hasn't turned up anything especially useful.

Reply:
“>” at the beginning of the paragraph should make it a blockquote.

Reply:
Ah... I like the look of that, but always wondered if that wasn't a misuse of blockquotes. When I see that, I always assume I am looking at an actual quote.

Reply:
Thanks! Converted to a blockquote.